CFEScraping/index.js

```diff
@@ -0,0 +1,13 @@
+const puppeteer = require('puppeteer');
+
+(async () => {
+    const browser = await puppeteer.launch();
+    const page = await browser.newPage();
+
+    await page.goto('https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/');
+    console.log('Successful');
+    await page.waitForTimeout(5000);
+    await page.screenshot({ path: 'Cfe.png' });
```

Comment:
Pls create a particular folder & save the images there, ( maybe we will not need them ), Same as in line 7, use a const or env var

Reply:
After creating the folder make sure never committed, so add it to the .gitignore